pkg/cluster/internal/providers/podman/provision.go

```diff
@@ -281,7 +281,7 @@ func getProxyEnv(cfg *config.Cluster) (map[string]string, error) {
 }
 
 func getSubnets(networkName string) ([]string, error) {
-	format := `{{range (index (index . "IPAM") "Config")}}{{index . "Subnet"}} {{end}}`
+	format := `{{ range (index (index (index (index . "plugins") 0 ) "ipam" ) "ranges")}}{{ index ( index . 0 ) "subnet" }} {{end}}`
```

Comment:
This format is getting too complicated now. might be worth just unmarshalling the JSON ourselves.
but fine as a follow up.

Also, is this after podman 3.0 ?
do we want to warn/error if podman version is less than that.

@aojea how is our Github Actions working, is this path not exercised?

Reply:
> This format is getting too complicated now. might be worth just unmarshalling the JSON ourselves.
but fine as a follow up.

amen, it took me a big chunk of time to come up with the template

> Also, is this after podman 3.0 ?
> do we want to warn/error if podman version is less than that.

I can't know if this worked before because this is only used to populate the proxy environment variables ... let's keep it this way, I can't imagine we have such power users of podman until now

> @aojea how is our Github Actions working, is this path not exercised?

nope

Reply:
+1 to json in a follow up